test/elixir_console_web/features/console_test.exs

```diff
@@ -41,4 +41,17 @@ defmodule ElixirConsoleWeb.ConsoleTest do
       |> assert
     end)
   end
+
+  feature "visitor can shortcut used expressions", %{session: session} do
```

Comment:
I don't find the word `shortcut` very descriptive of what the up/down arrows do.
Maybe something like "visitor can cycle through previously used commands" ?
wdyt?

Reply:
What do you all think about making this test to better cover this feature, for example having more than one command in the history and using both up and down arrows to cycle through more previous entries?